src/js/components/covid19/recipient/RecipientSection.jsx

```diff
@@ -21,16 +27,24 @@ const RecipientSection = () => {
     return (
         <div className="body__content recipient__container">
             <DateNote />
-            <h3 className="body__narrative">
-                <strong>Who</strong> received funding through COVID-19 awards?
-            </h3>
+            {arpStyles ?
+                <h3 className="body__narrative">
+                    <strong>Who</strong> received funding through American Rescue Plan awards?
+                </h3> :
+                <h3 className="body__narrative">
+                    <strong>Who</strong> received funding through COVID-19 awards?
+                </h3>
+            }
             <div className="body__narrative-description">
                 <p>
                     Once the federal government has determined that an individual, organization, business, or state, local, or tribal government will receive an award, the money is obligated (promised) and then outlayed (paid) according to the terms of the contract or financial assistance.<sup>3</sup>
```

Comment:
Unrelated to your changes, so this is an optional fix. I think this should be a 1 instead of a 3
```suggestion
                    Once the federal government has determined that an individual, organization, business, or state, local, or tribal government will receive an award, the money is obligated (promised) and then outlayed (paid) according to the terms of the contract or financial assistance.<sup>1</sup>
```